app/assets/stylesheets/layout.scss

```diff
@@ -0,0 +1,13 @@
+@import "foundation_and_overrides";
+@import "globals/all";
+
+.off-canvas-wrap,
+.inner-wrap {
+  min-height: 100%;
+}
+
+.off-canvas-wrap {
+  height: 100%;
+  overflow-y: scroll !important;
```

Comment:
!important should not be used<br>Properties should be ordered height, -webkit-overflow-scrolling, overflow-y